source/User_Guide/Marketing_Campaigns/senders.md

```diff
@@ -1,84 +1,60 @@
-* **Reply-To Email Address** - If your user hits reply in their email, the reply will go to this address.
-* **Company Address, City, State, Zip Code, Country** - The address of your business, for CAN-SPAM compliance.
-* **Nickname** - This is the name of this sender identity, which can be useful for identifying this identity in your list of senders. It will not be visible to your recipients.
-
-Once you fill out this information and hit “save,” you will then be able to verify your sender.
-
-{% anchor h2 %}
-Verifying Your Sender Identity
-{% endanchor %}
-
-We will not allow you to send with a sender identity that is not verified. This is a security feature that helps you be in compliance with [CAN-SPAM](http://www.business.ftc.gov/documents/bus61-can-spam-act-compliance-guide-business) when you send through SendGrid.
-
-To verify your identity, create the identity following the instructions above. Once you click “save,” we will send an email to the address you specified.
-
-{% info %}
-Your verification link is only valid for 48 hours. After that time you will need to start the verification process over.
-{% endinfo %}
-
-1. Check your email for the verification email we sent.
-2. Click the link provided in the verification email.
-3. Your sender identity is now verified!
-
-To resend your verfication email, click the gear icon for this sender and select 'view' from the menu.
+1. Click "Create New Sender"
+1. Fill out the entire "Add Sender Identity" form. This form is so extensive so that you are CAN-SPAM compliant. The footer of your emails includes the information you provide here.
+
+   * **From Name** - This is user-friendly name that is displayed to the user when they receive their email.
+   * **From Email Address** - This will display to the user as the email address who sent this email.
+   * **Reply-To Email Address** - If your user hits reply in their email, the reply will go to this address.
+   * **Company Address, City, State, Zip Code, Country** - The address of your business, for CAN-SPAM compliance.
+   * **Nickname** - This is the name of this sender identity, which can be useful for identifying this identity in your list of senders. It will not be visible to your recipients.
+   ![]({{root_url}}/images/sender_identity_1.png "Sender Identities")
+
+   {% warning %}
+   You must verify your sender identity before you can edit it or use it to send a campaign.
+   {% endwarning %}
+
+   {% warning %}
+   When you schedule a campaign, the sender identity used will not be able to be deleted.
+   {% endwarning %}
+
+1. After you fill out the form, click "Save".
+1. Check the email account and click the link in the email to verify the Sender email.
+   
+   {% info %}
+   Your verification link is only valid for 48 hours. After that time you will need to restart the verification process.
+   {% endinfo %}
+   
+   * To resend your verification email, click the gear icon for this sender and select 'view' from the menu.
```

Comment:
"To resend your verification email, click the gear icon under Actions on the Sender Management page and select **Resend Verification**.